.github/workflows/periodic.yml

```diff
@@ -0,0 +1,67 @@
+name: periodic
+
+on:
+  schedule:
+    # We have several schedules so jobs can check github.event.schedule to activate only for a fraction of the runs.
```

Comment:
I guess you copy this schedule from PyTorch periodic.  That workflow has several parts that need to be run with different schedules.  ET doesn't need that here, so let's keep only one schedule.  PyTorch periodic workflow originally started with every 4 hours, maybe that's a good start or it could be a longer period depending on how frequent a new commit lands in ET trunk https://hud.pytorch.org/hud/pytorch/executorch/main